pulsar/consumer_impl.go

```diff
@@ -72,6 +73,14 @@ func newConsumer(client *client, options ConsumerOptions) (Consumer, error) {
 		return nil, newError(SubscriptionNotFound, "subscription name is required for consumer")
 	}
 
+	if options.PriorityLevel <= 0 {
+		options.PriorityLevel = 0
```

Comment:
It should be return error. Ref: https://github.com/apache/pulsar/blob/master/pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerBuilderImpl.java#L254

Reply:
ok, i will add then error later